python/common/business-registry-model/src/business_model/models/action.py

```diff
@@ -0,0 +1,44 @@
+# Copyright © 2025 Province of British Columbia
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""This module holds data for actions."""
+
+from datetime import datetime
+
+from .db import db
+
+
+class Action(db.Model):
+    """This class manages all of the actions."""
+
+    __tablename__ = 'actions'
+
+    id = db.Column(db.Integer, primary_key=True)
+    action_name = db.Column('action_name', db.String(100), nullable=False, unique=True)
+    description = db.Column('description', db.String(255), nullable=True)
+    created_date = db.Column('created_date', db.DateTime(timezone=True), default=datetime.utcnow)
+    last_modified = db.Column('last_modified', db.DateTime(timezone=True), default=datetime.utcnow)
```

Comment:
```suggestion
    created_date = db.Column('created_date', db.DateTime(timezone=True), default=func.now())
    last_modified = db.Column('last_modified', db.DateTime(timezone=True), default=func.now())
```
Do the same update in other models as well.

ref: https://github.com/bcgov/lear/blob/main/python/common/business-registry-model/src/business_model/models/business.py#L252